### PR TITLE
Feature/feedme allow reminders

### PR DESCRIPTION
**Pull request by christoph-holter**
- you can "subscribe" to a foodtype (Fabrik, Erni's, Spar, Billa, Burgerking, Gumpendorfer or Pizza) by `foodtype +{integer}`, e.g. `ernis +1`. Note: Only your name will be added to the subscriber list as hubot can't possibly know which person is meant by `ernis +2`.
- on default food-time (11:45) all subscribed users will be notified in `#vie-food` (and automagically unsubscribed as they're expected to be fed) 
- you can set the reminder-time with `foodtype time`, e.g. `ernis 12:00`. The old reminder will be cleared and a new will be set. 
- version bump to 0.2.0

fixes:
- change output from fabrik to show correct weekend behaviour
- move date calc to `feedme` command.

### CODE REVIEW

File: scripts/feedme.coffee [hunk 1]
```diff
@@ -104,15 +109,78 @@ module.exports = (robot) ->
...
+      # clear all eaters, they're supposed to be fed.
+      delete eater[foodType]
+      robot.brain.set "feedme.eater", eater
+    ), moment().hour(hour).minute(minute).seconds(0)
+      .diff(moment(), 'milliseconds')
+
+  existsReminder = (foodType) ->
+    foodType of reminder
+
+  # clear all reservations + reminders @midnight
```
Comment (lorenzleutgeb) [inline scripts/feedme.coffee:148]: Not sure this is stable enough. Why not clear right after sending out the reminder?

File: scripts/feedme.coffee [hunk 2]
```diff
@@ -104,15 +109,78 @@ module.exports = (robot) ->
...
+
     res.send "Heutiges Mittagsmenü:\n
       \nFabrik: \n#{fabrikMenu}\n
-      \nErni's: \n#{ernisMenu}"
+      \nErni's: \n#{ernisMenu}\n
+      \nNatürlich gibt es auch noch:
+      \nSpar, Billa, Burgerking, Gumpendorfer oder Pizza."
+
+  # subscribe to food and reminder
+  robot.hear /(fabrik|ernis|spar|billa|burgerking|gumpendorfer|pizza) \+\d*/i
```
Comment (lorenzleutgeb) [inline scripts/feedme.coffee]: I'd prefer matching on unique prefixes, e.g. `/(fa?b?r?i?k?|er?n?i?s?|sp?a?r?|bil?l?a|bur?g?e?r?k?i?n?g?)/` and so on. Also, we could include common abbreviations that are not substrings of the regex like `"bk"`. Furthermore this regex is partly duplicated on line 179. We should define it only once.

File: scripts/feedme.coffee (hunk 1 above)
Comment (christoph-holter) [inline scripts/feedme.coffee:148]: This is already the case. Look at :128 and :130. As there can be a day without reminder I wanted to make sure to clear all subscribers on midnight anyway to have no leftovers on the next day

File: scripts/feedme.coffee [hunk 3]
```diff
@@ -12,6 +12,20 @@ urllib = require "urllib-sync"
 jsdom = require "jsdom"
 moment = require "moment"
 
+FOOD_TIME =
+  HOUR : 11
+  MINUTE : 45
+
+FOOD_REGEX = "fab?r?i?k?|ern?i?s?|spa?r?|bil?l?a?|bur?g?e?r?k?i?n?g?|bk|gum?p?e?n?d?o?r?f?e?r?|piz?z?a?"
```
Comment (capsur) [inline scripts/feedme.coffee]: How do we call the asia thingy @Margartengürtel? Should we add that?

**Pull request merged**